pages/library/viewBook.js

```diff
@@ -107,11 +116,15 @@ export default function ViewBook({ library, currentBook }) {
                   .filter((wisdom) => wisdom.category === cat)
                   .map((wisdom) => (
                     <StyledWisdom key={wisdom.id}>
-                      <h2>Question:</h2>
-                      <p>{wisdom.question}</p>
+                      <StyledLink href={`/library/${wisdom.id}`}>
+                        <div>
+                          <h2>Question:</h2>
+                          <p>{wisdom.question}</p>
 
-                      <h2>Answer:</h2>
-                      <p>{wisdom.answer}</p>
+                          <h2>Answer:</h2>
+                          <p>{wisdom.answer}</p>
+                        </div>
+                      </StyledLink>
 
                       <div className="seperator">
```

Comment:
This is not part of the PR, but 🫣:
Why are you declaring a class here?
otherwise you used styled components and here you don't. you might want to use a consistent method for declaring components to keep the code readable and easy to maintain.

Reply:
This was a recommandation for the previous pr, not my idea... for accessability reasons🤷

Reply:
Okay, then I did not say anything😅

Overall, fantastic job!